File: src/server/InvestmentApp-Server/Models/Experts/Expert.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace InvestmentApp.Models.Experts;

public class Expert
{
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public Guid Id { get; set; }

    public string Name { get; set; }

    public string SurName { get; set; }

    public string MiddleName { get; set; }

    public double CompetenceCoefficient { get; set; }

    public string WorkPlace { get; set; }

    public string Specialty { get; set; }
}

File: src/client/src/app/models/expert.ts
import { DataTableRow, Role } from ".";
import { Guid } from "guid-typescript";

export interface Expert {
    id: Guid
    name: string;
    surName: string;
    middleName: string;
    wokPlace: string;
    speciality: string;
    competenceCoefficient: number;
}

export interface ExpertRow extends DataTableRow {
    id: string,
    internalId: string;
    name: string;
    surName: string;
    middleName: string;
    wokPlace: string;
    speciality: string;
    competenceCoefficient: number;
}
